packages/manager/modules/pci/src/projects/project/analytics-data-platform/onboarding/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,11 @@
+{
+  "analytics_data_platform_onboarding_content1": "Déployer un cluster big data est un processus long et contraignant. OVH Analytics Data Platform vous simplifie votre métier. En moins d’une heure, nous vous livrons une stack Apache Hadoop, préconfigurée et prête à l’usage.",
+  "analytics_data_platform_onboarding_content2": "En nous basant sur un standard de distribution Hadoop open source, nous préconfigurons tous les services nécessaires à vos traitements de données et sécurisons vos flux avec le monde externe ainsi que vos utilisateurs.",
+  "analytics_data_platform_onboarding_content3": "Déployez OVH Analytics Data Platform pour de nombreux usages : l'analyse des marchés, l'informatique décisionnelle, l’IoT ou encore la maintenance prévisionnelle. À vous de jouer !",
+  "analytics_data_platform_onboarding_action_label": "Créer mon cluster Analytics Data Platform",
+
+  "analytics_data_platform_onboarding_guides_overview_title": "Vue d'ensemble",
+  "analytics_data_platform_onboarding_guides_retrieve-credentials_title": "Récupérer les informations d'identification",
```

Comment:
To keeps things organized it would be great to stick with snakeCase for translation key.

Reply:
This is fixed